lib/utils/npm.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const _chalk = require('chalk')
+const _exec = require('child-process-promise').exec
+const _Promise = require('promise')
+
+module.exports = {
+  /**
+   * Run an npm command.
+   * @param {String} command the name of the npm command (view, install, unistall, etc.)
+   * @param {Array} args a list of arguments
+   * @returns {Promise} a promise that will return the command result once it's run
+   */
+  _run: function (command, args = []) {
```

Comment:
We can run a command in `ember-cli` but we are not able to catch the output of the command. In the `view` case we need the output of the command.